infra/terraform/modules/_auth/secrets.tf

```diff
@@ -0,0 +1,14 @@
+resource "aws_secretsmanager_secret" "key_pair" {
+  name                    = "${var.name}/key-pair"
```

Comment:
if this name could have a random string added to the end, it would solve the issue of 
'You can't perform this operation on the secret because it was marked for deletion.'